pkgs/development/tools/rust/racer/default.nix

```diff
@@ -1,22 +1,48 @@
-{ lib, rustPlatform, fetchCrate, makeWrapper, stdenv, Security }:
+{ lib, pkgs, rustPlatform, fetchFromGitHub, makeWrapper, stdenv, Security }:
 
-rustPlatform.buildRustPackage rec {
+let rustPinnedChannelOverlay = final: prev:
+  let
+    # as per https://github.com/racer-rust/racer/issues/1181#issuecomment-1099371055
+    rustChannel = prev.rust-bin.nightly."2022-04-06";
+  in
+  {
+    inherit rustChannel;
+    rustc = rustChannel.default.override {
+      # build process requires ast tools
+      extensions = [ "rust-src" "rustc-dev" "llvm-tools-preview" ];
+    };
+  };
+in
+with import <nixpkgs>
+{
+  overlays = [
+    (import (fetchTarball "https://github.com/oxalica/rust-overlay/archive/master.tar.gz"))
+    rustPinnedChannelOverlay
+  ];
+};
```

Comment:
Sorry but we cannot have overlays importing overlays from external resources. Hydra does not allow to fetch from this.

Since this project is no longe developed, should we maybe drop it from nixpkgs? https://github.com/racer-rust/racer#disclaimer

Reply:
Thanks for the feedback @Mic92 - I'm new to this so was not aware of the overlay constraint; I was just focused on getting the package built.

Yep - removing it makes sense; I can modify this PR accordingly.